x509/cert_pool.go

```diff
@@ -119,6 +119,20 @@ func (s *CertPool) AppendCertsFromPEM(pemCerts []byte) (ok bool) {
 	return
 }
 
+// Contains returns true if c is in s.
+func (s *CertPool) Contains(c *Certificate) bool {
+	candidates, ok := s.byName[string(c.RawSubject)]
+	if !ok {
+		return false
+	}
+	for _, candidateNum := range candidates {
```

Comment:
I don't want indicies of this array, I want the values of this array, which happen to be indicies for a different array.